jerry-core/config.h

```diff
@@ -575,6 +579,10 @@
 || ((JERRY_BUILTIN_SHAREDARRAYBUFFER != 0) && (JERRY_BUILTIN_SHAREDARRAYBUFFER != 1))
 # error "Invalid value for JERRY_BUILTIN_SHAREDARRAYBUFFER macro."
 #endif
+#if !defined (JERRY_BUILTIN_ATOMICS) \
```

Comment:
I would use the JERRY_BUILTIN_SHAREDARRAYBUFFER macro to enable atomics, otherwise the shared array buffer is pointless. What others think?